src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -467,14 +467,30 @@ void Replayer<I>::handle_load_remote_group_snapshots(int r) {
   if (is_replay_interrupted()) {
     return;
   }
-
-  auto last_remote_snap = m_remote_group_snaps.rbegin();
   if (r < 0) {  // may be remote group is deleted?
     derr << "error listing remote mirror group snapshots: " << cpp_strerror(r)
          << dendl;
     handle_replay_complete(r, "failed to list remote group snapshots");
     return;
-  } else if (is_resync_requested()) {
+  }
+  for (auto snap = m_remote_group_snaps.begin(); snap!= m_remote_group_snaps.end();) {
```

Comment:
Suggestions:
1. leave a space after ` != `
2. may be use variable name as `remote_snap` for better readability instead of `snap`?

Reply:
> 1. leave a space after `!=`

... and, if fixing spacing anyway, after the second semicolon (i.e. before the loop's closing paren).